recipes/icechunk/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "icechunk" %}
+{% set version = "0.1.0a15" %}
```

Comment:
@andersy005 the updates here made my comment "outdated," but this is still a dev/unstable release and must follow CFEP-05.

See https://github.com/conda-forge/cfep/blob/main/cfep-05.md

Reply:
there is now a `v0.1.0` release. i just updated the recipe